CHANGELOG.md

```diff
@@ -7,11 +7,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.1.1] - 2020-01-20
```

Comment:
Following semantic versioning this says that: hey, we fixed one or few things only. But we don't, we added functionality (in build), so it would be `0.2.0` (but not now).